db/schema.rb

```diff
@@ -54,6 +54,7 @@
     t.string   "label",      null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string   "slug"
```

Comment:
Given we're using this field for lookups it will be useful if we have a unique index on it to make that faster.  It might also be good to have it as a non-null column while we're at it.  We can back those up with the relevant validations and specs too.  Doing all that means we'll want to add some extra code to the migration as well to make sure any existing rows won't be invalid.

Something like:

```ruby
add_column :services, :slug, unique: true

Service.all.each do |service|
  service.slug = service.label.parameterize
  service.save!
end

change_column_null :services, :slug, false
```

Although it's worth reading this [blog post about using models in migrations](https://coderwall.com/p/zav1dg/using-models-in-rails-migrations) too, as there are reasons not to just use `Service` directly (tl;dr so that the migrations can run in a future world where `Service` has been renamed or removed).